NEWS

```diff
@@ -68,6 +68,12 @@ Major new features:
 Changes in behavior compared to prior versions:
 
 Removed legacy support:
+- "yoda" SPML component is removed from the Open SHMEM codebase. It was utilizing
+  Open MPI BTL components for supported networks.
+  Currently avalable SPML components are:
```

Comment:
I can do that, My intention was to provide alternatives for those who was relying on yoda.